brd-ios/breadwallet/src/Redesign/Scenes/RegistrationConfirmation/RegistrationConfirmationPresenter.swift

```diff
@@ -58,6 +58,8 @@ final class RegistrationConfirmationPresenter: NSObject, Presenter, Registration
     
     func presentValidate(actionResponse: RegistrationConfirmationModels.Validate.ActionResponse) {
         let code = actionResponse.item ?? ""
+        guard code.count <= 6 else { return }
```

Comment:
this is interactor logic. you should do this in interactor and pass  the bool value to presenter